hq/app/config/Config.scala

```diff
@@ -67,20 +67,6 @@ object Config {
     )
   }
 
-  def gcpSccAuthentication(implicit config: Configuration): GcpSccConfig = {
-    val gcpOrgId = requiredString(config, "gcp.orgId")
-    val gcpSccSourceId = requiredString(config, "gcp.sccSourceId")
-    GcpSccConfig(gcpOrgId, gcpSccSourceId)
-  }
-
-  def gcpCredentialsProvider(implicit config: Configuration): FixedCredentialsProvider = {
```

Comment:
@mchv we will need to remove the credentials from the Google Developer Console - I believe you have the permissions to do this?

Reply:
@mchv The credentials can now be deleted - thank you!